CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ Please visit [cucumber/CONTRIBUTING.md](https://github.com/cucumber/cucumber/blo
 
 ### Changed
 
+- Add Ruby 3.2 to the CI matrix. ([PR#1690](https://github.com/cucumber/cucumber-ruby/pull/1690))
```

Comment:
Essentially updating the changelog would be unnecessary for this change. Because this is a CI update to the internal development of this gem and has no impact on users.

Reply:
I followed choice when added 3.1 below, but it may indeed be unnecessary. I updated this PR.
https://github.com/cucumber/cucumber-ruby/blob/b9a7b01ec7f39a6d950094c4a2f9b4c178e690fb/CHANGELOG.md?plain=1#L59